vocabs/DID-core-shape-v1.shex

```diff
@@ -0,0 +1,39 @@
+# This shape expression is hosted at  https://www.w3.org/ns/did/shex/v1
+#
+# You can run this (using https://www.npmjs.com/package/shex/v/0.5.5), for example, with:
+#  shex -x DID-core-shape-v1.shex -d shape_test.ttl -n did:ex:12345 -s SubjectShape
+# 
+
+#
+PREFIX did:  <https://www.w3.org/ns/did/vocab/v1#>
```

Comment:
```suggestion
PREFIX did:  <https://www.w3.org/ns/did#>
PREFIX sec:  <https://w3id.org/security#>
```

Reply:
@msporny 

> @iherman what happens if you put a `did.var` file in the `/ns/` directory to purely direct it to the HTML description?

tried it, ain't working. It seems that the MultiViews feature ignores the `did.var` file if a `did` directory name is also present. This is exactly the problem. I tried to look around the web for solutions, did not find anything working. There were some proposals to try to switch MultiViews off, but this is not something I should do in the `/ns` directory, because it may have consequences on other things in the same directory. Ie, that is a big no-no.